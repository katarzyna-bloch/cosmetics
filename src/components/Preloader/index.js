import React from 'react'
import styled from 'styled-components'

const Image = styled.img`
  height: 100px;
`

const Preloader = () => (
  <Image src="https://veganvita24.pl/kalkulator/img/Loading.gif" />
)

export default Preloader
