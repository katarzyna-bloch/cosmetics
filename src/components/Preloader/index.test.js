import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Preloader from '.'

it('should render correctly', () => {
  const wrapped = shallow(<Preloader />)
  expect(toJson(wrapped)).toMatchSnapshot()
})
