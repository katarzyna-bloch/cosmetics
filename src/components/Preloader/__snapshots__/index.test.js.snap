// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<styled.img
  src="https://veganvita24.pl/kalkulator/img/Loading.gif"
/>
`;
